topics/ecology/tutorials/Biodiversity data exploration/tutorial.md

```diff
@@ -40,255 +46,231 @@ This tutorial will guide you on the exploration of biodiversity data having taxo
 >
 {: .agenda}
 
-# Downloading biodiversity data
+# Downloading and pre-processing biodiversity data
 
 First step is to download biodiversity data on your Galaxy history. Here we will use a "classical" (containing taxonomic, spatial and temporal informations) biodiversity dataset from the well known ["Reef life survey" initiative](https://reeflifesurvey.com/).
 
 ## Get data
 
 > ### {% icon hands_on %} Hands-on: Data upload
 >
-> 1. Create a new history for this tutorial
+> 1. Create a new history for this tutorial and give it a name (example: "RLS for biodiversity data exploration tutorial")
+>    for you to find it again later if needed.
 > 2. Import the files from [Zenodo](https://zenodo.org/record/5930763/files/Reel_life_survey_fish_sample.tabular?download=1)
```

Comment:
Change the link here to "https://zenodo.org/record/6107457/files/IMOS-National_Reef_Monitoring_Network_Sub-Facility-Global_reef_fish_abundance_and_biomass.csv?download=1"